src/tests/CMakeLists.txt

```diff
@@ -53,16 +53,25 @@ find_package(Boost 1.57.0 REQUIRED COMPONENTS unit_test_framework)
   list(APPEND test_cases test_jobtopology)
   list(APPEND test_cases test_dipoledipoleinteraction)
   list(APPEND test_cases test_populationanalysis)
-list(APPEND test_cases test_gaussian)
-list(APPEND test_cases test_orca)
-list(APPEND test_cases test_nwchem)
-list(APPEND test_cases test_dftengine)
-list(APPEND test_cases test_bsecoupling)
-list(APPEND test_cases test_rate_engine)
+  list(APPEND test_cases test_gaussian)
+  list(APPEND test_cases test_orca)
+  list(APPEND test_cases test_nwchem)
+  list(APPEND test_cases test_dftengine)
+  list(APPEND test_cases test_bsecoupling)
+  list(APPEND test_cases test_rate_engine)
+  if(USE_GPU)
+    list(APPEND test_cases test_eigencuda)
```

Comment:
@junghans can this be expressed cleaner? I would preferably have less if(GPU) stuff over the code?

Reply:
Not sure what you have in mind.....

Reply:
no idea, that is why I am asking

Reply:
Seems pretty clean to me, alternatively you could make the test a noop for USE_CUDA=OFF and always run it.